src/main/Main.cpp

```diff
@@ -29,9 +29,28 @@
 #include <cstdlib>
 #include <exception>
 #include <iostream>
+#include <string_view>
 
 using namespace util::config;
 
+static int
+handleVerifyConfig(std::string_view configPath)
```

Comment:
Better to put it into a separate file and then you will be able to test it. Also I would call it `verifyConfig`. And better for it to return bool instead of exit code.